.bitmap

```diff
@@ -1122,11 +1122,23 @@
         "mainFile": "index.ts",
         "rootDir": "scopes/component/component-drawer"
     },
-    "ui/component-filters": {
-        "scope": "teambit.component",
-        "version": "0.0.10",
+    "ui/component-filters/deprecate-filter": {
+        "scope": "",
+        "version": "",
+        "mainFile": "index.ts",
+        "rootDir": "scopes/component/component-filters/deprecate-filter"
+    },
+    "ui/component-filters/env-filter": {
+        "scope": "",
+        "version": "",
+        "mainFile": "index.ts",
+        "rootDir": "scopes/component/component-filters/env-filter"
+    },
+    "ui/component-filters/use-component-filter": {
```

Comment:
why did you decide on calling it `use-component-filter`? its not really a hook. most of what's left there is the context.
so maybe `component-filter-context` or something like that?